app/config/services.php

```diff
@@ -18,6 +23,24 @@
  */
 
 $container = new ContainerBuilder();
```

Comment:
Bedoel je ```getMailTransport()```? Is die niet op het moment juist in de service ```getMailer()```? Aangezien ze allebei in een container zijn, kan de getMailer() de getMailTransport() het gewoon gebruiken, niet?

Of bedoel je dat die er zo in de container is gezet, terwijl je het van buitenaf in de container wilt injecteren?